src/main/java/tech/toparvion/analog/service/choice/ConditionalOnChoicesAutoReloadEnabled.java

```diff
@@ -1,13 +1,19 @@
 package tech.toparvion.analog.service.choice;
 
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Conditional;
 
-import java.lang.annotation.*;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
 
 import static java.lang.annotation.ElementType.TYPE;
 import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
 /**
+ * Custom {@link Conditional @Conditional} that checks if choices <code>auto reload</code> enabled in <code>properties</code>.<br/>
+ * This is some kind of "syntax sugar" witch is designed to add this condition to all required places.
```

Comment:
```suggestion
 * This is some kind of "syntax sugar" which is designed to add this condition to all required places.
```

Because "[witch](https://m.media-amazon.com/images/I/41wGcAsNeOL.jpg)" is not exactly what you'd want to say 😉